src/api/handlers/deviceprofile.py

```diff
@@ -95,17 +96,18 @@ def platform_metrics(self, request):
     metric = args.get("metric", None)
     community_id = args.get("community_id", None)
     
-    if metric == "anonymous_users":
-      metric, err = self.service.metric_anonymous_users(context, args)
+    if metric:
```

Comment:
Question - if metric is passed in, but not one of the expected choices, I think that should be an error.  Pass back InvalidResourceError perhaps?

Reply:
Sounds good